README.md

```diff
@@ -167,13 +167,114 @@ wrapping a component you intend to style, you'll need to make sure you accept
 the `className` as a prop and apply it to where you want the styles applied in
 your custom component (normally the root element).
 
-##### ...styles
```

Comment:
`...styles` is an argument to the `glamorousComponentFactory` which is why it had the additional `#` in here (to be a sub-child of that section).